modules/cert-manager-configure-cloud-credentials-aws-sts.adoc

```diff
@@ -0,0 +1,124 @@
+
+.Prerequisites
+
+* You extracted and prepared the `ccoctl` binary.
+* You have configured an {product-title} cluster with AWS STS by using the Cloud Credential Operator in manual mode.
+
+.Procedure
+
+. Create a directory to store a `CredentialsRequest` resource YAML file by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file under the `credentials-request` directory, such as, `sample-credential-request.yaml` by applying the following yaml:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "route53:GetChange"
+      effect: Allow
+      resource: "arn:aws:route53:::change/*"
+    - action:
+      - "route53:ChangeResourceRecordSets"
+      - "route53:ListResourceRecordSets"
+      effect: Allow
+      resource: "arn:aws:route53:::hostedzone/*"
+    - action:
+      - "route53:ListHostedZonesByName"
+      effect: Allow
+      resource: "*"
+  secretRef:
+    name: aws-creds
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
+
+. Use the `ccoctl` tool to process `CredentialsRequest` objects by running the following command:
++
+[source,terminal]
+----
+$ ccoctl aws create-iam-roles \
+    --name <user_defined_name> --region=<aws_region> \
+    --credentials-requests-dir=<path_to_credrequests_dir> \
+    --identity-provider-arn <oidc_provider_arn> --output-dir=<path_to_output_dir>
+----
++
+.Example output
+[source,terminal]
+----
+2023/05/15 18:10:34 Role arn:aws:iam::XXXXXXXXXXXX:role/<user_defined_name>-cert-manager-aws-creds created
+2023/05/15 18:10:34 Saved credentials configuration to: <path_to_output_dir>/manifests/cert-manager-aws-creds-credentials.yaml
+2023/05/15 18:10:35 Updated Role policy for Role <user_defined_name>-cert-manager-aws-creds
+----
++
+Copy the `<aws_role_arn>` from the output to use in step 7. For example, `"arn:aws:iam::XXXXXXXXXXXX:role/<user_defined_name>-cert-manager-aws-creds"`
+. Add the `eks.amazonaws.com/role-arn="<aws_role_arn>"` annotation to the service account by running the following command:
++
+[source,terminal]
+----
+$ oc -n cert-manager annotate serviceaccount cert-manager eks.amazonaws.com/role-arn="<aws_role_arn>"
+----
+
+. To create a new pod, delete the existing cert-manager controller pod by running the following command:
++
+[source,terminal]
+----
+$ oc delete pods -l app.kubernetes.io/name=cert-manager -n cert-manager
+----
++
+A new cert-manager controller pod is updated with the AWS credentials after a couple of seconds.
```

Comment:
```suggestion
The AWS credentials are applied to a new cert-manager controller pod within a couple of seconds.
```
Also reconsider whether `couple of seconds` is required because it's a negligible duration.